docs/ops/detection/Proposal_4.md

```diff
@@ -0,0 +1,160 @@
+## Proposal <a name="Proposal"></a>
+
+**Versioned name**: *Proposal-4*
+
+**Category**: *Object detection*
+
+**Short description**: *Proposal* operation filters bounding boxes and outputs only those with the highest prediction confidence.
+
+**Detailed description**
+
+*Proposal* has three inputs: a tensor with probabilities whether particular bounding box corresponds to background and foreground, a tensor with logits for each of the bounding boxes, a tensor with input image size in the [`image_height`, `image_width`, `scale_height_and_width`] or [`image_height`, `image_width`, `scale_height`, `scale_width`] format. The produced tensor has two dimensions `[batch_size * post_nms_topn, 5]`.
```

Comment:
Here the last statement is that the operation produces one tensor. It should be updated

Reply:
Agree. In the last version updated this sentence.